save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectDemoMenu.kt

```diff
@@ -176,11 +180,15 @@ val projectDemoMenu: FC<ProjectDemoMenuProps> = FC { props ->
                 div {
                     className = ClassName("flex-wrap d-flex justify-content-around")
                     when (demoStatus) {
-                        DemoStatus.NOT_CREATED -> buttonBuilder("Create") {
-                            sendDemoCreationRequest()
+                        DemoStatus.NOT_CREATED -> if (getHighestRole(props.organizationRole, props.projectRole).isHigherOrEqualThan(Role.OWNER)) {
+                            buttonBuilder("Create") {
+                                sendDemoCreationRequest()
+                            }
                         }
-                        DemoStatus.STARTING -> buttonBuilder("Reload") {
-                            getDemoStatus()
+                        DemoStatus.STARTING -> if (props.projectRole.isHigherOrEqualThan(Role.VIEWER)) {
```

Comment:
I guess organization owner should be able to do the same

Reply:
To my mind those buttons should be present, but disabled. This is what I written in issue. Please remove `if` from here and add `disabled` to buttons.

Reply:
and probably need to add a hint message on button, that it's disabled due to lack of permissions